### PR TITLE
Modern Templates for Issue and Pull Requests

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
Issue/bug report templates were in markdown format. This PR will update that template's format to .yml format.
Also, a separate feature request template is added.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (sepandhaghighi) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: @sadrasabouri Why did you remove this file?
Please revert the change.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    validations:
+      required: false
+  - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional context
+      placeholder: >
+        Add any other context or screenshots about the feature request here.
+    validations:
+      required: false
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/feature_request.yml:36]: @sadrasabouri 
Please address this minor end-of-file bug.

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (sadrasabouri) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: fixed in [84a48df](https://github.com/sepandhaghighi/nafas/pull/89/commits/84a48df03ca7230398916abfe4a6a1e5ea046726).

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 1 above)
Comment (sadrasabouri) [reply .github/ISSUE_TEMPLATE/feature_request.yml]: fixed in [5a9076a](https://github.com/sepandhaghighi/nafas/pull/89/commits/5a9076ae62e0b1eb0dd3fef15ec8c60ba6ccacd4)

**Review by sepandhaghighi — APPROVED** [review]
🚀
